src/CodeGenerator/src/Generator/GeneratorHelper.php

```diff
@@ -111,6 +113,7 @@ public static function parseDocumentation(string $documentation, bool $short = t
         $spaceNext = false;
 
         // converts <li> into `- ` AND handle multi-level list
+        $counter = null;
```

Comment:
this way of handling the counter does not seem to be compatible with nested lists (while the comment on the previous line says we support it)

Reply:
good catch.